### ISSUE TITLE
Bug: Luban distorts SVG files when importing (Laser, CNC)

### ISSUE THREAD

**Issue by ProfCampbokk** [Snapmaker/Luban#1877]
# 🐞 bug report

### Affected Version(s)
Visible in 4.4.0 and 4.5.0

### Is this a regression? (optional)

 I'm not aware of this working previously.

### To Reproduce
1. Go to 2D laser mode
2. Open my SVG file
3. Observe the distortion
 A clear and concise description of the problem...

Exception or Error (optional)
<pre><code>

N/A
</code></pre>

Expected behavior

The file should load identically to how it looks in other software, e.g. Lightburn or Inkscape.

## 🌍  Your Environment

**Platform:**
<pre><code>

- Operating System: [macOS 13.1 (22C65)]
- Printer: [Not relevant]

</code></pre>

![dividers-cut-1](https://user-images.githubusercontent.com/98917104/212184452-ea93b5ae-f317-4677-8649-1be9844dc440.svg)

https://user-images.githubusercontent.com/98917104/212184597-40b62e9a-26a2-4c7e-a9b3-b8f33d988a27.mov

**Comment by ProfCampbokk** [comment]
Also mentioned in this issue: https://github.com/Snapmaker/Luban/issues/1884

**Comment by parachvte** [comment]
This issue is different from #1884 

```
<tspan
           sodipodi:role="line"
           id="tspan1004"
           style="font-size:8.46667px;text-align:center;text-anchor:middle;fill:#0000ff;stroke-width:0.264583"
           x="36.953468"
           y="11.796615">North America</tspan>
```

It's caused by the `text-anchor:middle` style of `<tspan>`. You can change the anchor of text on your SVG editor to solve this problem temporarily.

**Comment by sumibin** [comment]
![image](https://user-images.githubusercontent.com/34013829/220497471-9997316d-f495-4572-8984-fadf6433ebdf.png)

fixed in v4.6.2

**Comment by maik-h** [comment]
Hallo zusammen ich habe das gleiche Problem bei der Version 4.11.0
Gibt es dort schon eine lösung?
![Screenshot 2024-04-28 152159](https://github.com/Snapmaker/Luban/assets/168349066/7c47cb9c-10ab-4945-a4e3-06ab83e25818)
![Screenshot 2024-04-28 152113](https://github.com/Snapmaker/Luban/assets/168349066/927e7e5f-6649-4c93-9174-5a583351e001)
